docs/modules/ROOT/pages/runbooks/CephMgrPrometheusModuleInactive.adoc

```diff
@@ -4,9 +4,9 @@ include::partial$runbooks/contribution_note.adoc[]
 
 == icon:glasses[] Overview
 
-The MGR/Prometheus module is unreachable.
-This could mean that the module has been disabled or the MGR itself is down.
-Without the MGR/Prometheus module metrics and alerts will no longer function.
+The mgr/prometheus module at {{ $labels.instance }} is unreachable.
```

Comment:
Having alert templates in the runbook is pointless.

```suggestion
The mgr/prometheus module is unreachable.
```

Reply:
True, somehow i was in PrometheuRule mode 🤷